draft-ietf-jsonpath-base.md

```diff
@@ -289,23 +293,25 @@ JSONPath expressions are applied to a JSON value, the *argument*.
 Within the JSONPath expression, the abstract name `$` is used to refer
 to the *root node* of the argument, i.e., to the argument as a whole.
 
-JSONPath expressions can use the *dot notation*
+JSONPath expressions primarily use the *bracket notation*
 
 ~~~~
-$.store.book[0].title
+$['store']['book'][0]['title']
 ~~~~
 
-or the more general *bracket notation*
+but may also use the shorthand *dot notation*
 
 ~~~~
-$['store']['book'][0]['title']
+$.store.book[0].title
 ~~~~
 
 to build paths that are input to a JSONPath implementation.
+Generally, the dot notation is preferred for most applications.
+Mixing these syntaxes within a single path is also useful.
 
```

Comment:
We definitely need some statement about allowing these syntaxes to be mixed.  Otherwise people might get the impression that they may only use one or the other (i.e. all dot syntax or all bracket syntax).

I don't mind the statement about preference, but I thought it would be good for general guidance.

Reply:
Ok, it's an editorial change rather than something substantive.